python/lsst/ts/scriptqueue/ui/request_cmd.py

```diff
@@ -0,0 +1,385 @@
+__all__ = ["RequestCmd"]
+
+import logging
+import warnings
+import argparse
+import yaml
+
+from lsst.ts.scriptqueue.utils import DETAIL_LEVEL
+try:
+    from cmd2 import Cmd
+except ModuleNotFoundError as e:
+    warnings.warn("Could not find module cmd2. Fallback to standard "
+                  "cmd library. This may limit the functionality of the shell.")
+    from cmd import Cmd
+
+from .request_model import RequestModel
+
+queue_names = ["Main", "Auxiliary"]
+
+
+class RequestCmd(Cmd):
+    """This class provides a command line interface to the RequestModel class.
+    """
+
+    def __init__(self, index):
+
+        self.intro = """Welcome to the Request Script shell. This script is designed to
+        operate with the LSST script queue.
+
+        Type help or ? to list commands."""
+        self.prompt = '(cmd): '
+
+        Cmd.__init__(self)
```

Comment:
This works, but it in Python 3 it's safer (more future proof) to use super: `super().__init__()`